modules/swagger-codegen/src/main/java/io/swagger/codegen/languages/GatlingCodegen.java

```diff
@@ -0,0 +1,345 @@
+package io.swagger.codegen.languages;
+
+import io.swagger.codegen.*;
+import io.swagger.models.*;
+import io.swagger.models.parameters.*;
+import io.swagger.models.properties.*;
+import org.apache.commons.io.FileUtils;
+import org.apache.commons.lang3.StringUtils;
+
+import java.io.IOException;
+import java.util.*;
+import java.io.File;
+
+public class GatlingCodegen extends AbstractScalaCodegen implements CodegenConfig {
+
+    // source folder where to write the files
+    protected String sourceFolder = "src" + File.separator + "gatling" + File.separator + "scala";
+    protected String resourceFolder = "src" + File.separator + "gatling" + File.separator + "resources";
+    protected String confFolder = resourceFolder + File.separator + "conf";
+    protected String dataFolder = resourceFolder + File.separator + "data";
+    protected String apiVersion = "1.0.0";
+
+    /**
+     * Configures the type of generator.
+     *
+     * @return the CodegenType for this generator
+     * @see io.swagger.codegen.CodegenType
+     */
+    public CodegenType getTag() {
+        return CodegenType.CLIENT;
+    }
+
+    /**
+     * Configures a friendly name for the generator.  This will be used by the generator
+     * to select the library with the -l flag.
+     *
+     * @return the friendly name for the generator
+     */
+    public String getName() {
+        return "gatling";
```

Comment:
@ryandavis84 what about naming it as "scala-gatling" instead?

Naming convention for new generator: "{lang}-{framework}"

Reply:
In other words, we'll also rename GatlingCodegen.java to ScalaGatlingCodegen.java

Reply:
I will get this done this weekend.  Thank you